### PR TITLE
Create autoyast installation without registration for minimal update qcow on SLES15SP3

### PR DESCRIPTION
**Pull request by lemon-suse**
We need create autoyast installation without registration for minimal update qcow on SLES15SP3.

- Related ticket: https://progress.opensuse.org/issues/130483
- Needles: N/A
- Verification run: 
create hdd: 
https://openqa.nue.suse.com/tests/11634061# x86_64 
https://openqa.nue.suse.com/tests/11634062#  aarch64
migration test:
http://openqa.nue.suse.com/tests/11623566#  x86_64
http://openqa.nue.suse.com/tests/11623732#  aarch64
- Related MR: https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/568

### CODE REVIEW

File: data/yam/support_images/autoyast_sles15sp3_media_install_textmode_minimal_aarch64.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0"?>
```
Comment (jknphy) [inline data/yam/support_images/autoyast_sles15sp3_media_install_textmode_minimal_aarch64.xml:1]: now there is intermediate /autoyast/ folder and the name of the .xml file doesn't require it.

**Review by jknphy — APPROVED** [review]

File: data/yam/support_images/autoyast_sles15sp3_media_install_textmode_minimal_aarch64.xml (hunk 1 above)
Comment (lemon-suse) [reply data/yam/support_images/autoyast_sles15sp3_media_install_textmode_minimal_aarch64.xml]: Ok, updated it and VR updated also.

**Comment by tinawang123** [comment]
Why does the file name include 'media'?

**Comment by lemon-suse** [comment]
> Why does the file name include 'media'?

This means installation through media (ISO).

File: data/yam/autoyast/support_images/sles15sp3_media_install_textmode_minimal_aarch64.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[dvd:///?devices=/dev/sr0]]></media_url>
+        <product>sle-module-serverapplications</product>
+        <product_dir>/Module-Server-Applications</product_dir>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[dvd:///?devices=/dev/sr0]]></media_url>
+        <product>sle-module-desktop-applications</product>
+        <product_dir>/Module-Desktop-Applications</product_dir>
```
Comment (chcao) [inline data/yam/autoyast/support_images/sles15sp3_media_install_textmode_minimal_aarch64.xml:33]: For this scenario: offline_sles15sp3_media_base_def_minimal@aarch64, is Desktop Applications Module necessary?

Comment (lemon-suse) [reply data/yam/autoyast/support_images/sles15sp3_media_install_textmode_minimal_aarch64.xml]: Thanks. This is not needed and forgot to remove. Updated.

**Pull request merged**